src/complexity_measures/sample_entropy.jl

```diff
@@ -0,0 +1,144 @@
+using DelayEmbeddings
+using Statistics
+using Neighborhood.NearestNeighbors: inrangecount, Chebyshev
+
+export SampleEntropy
```

Comment:
In the docstring of `complexity`, some methods are crossreffed, like `Dispersion`, and some are not, like `SampleEntropy`. What is the consensous? Either all should be corssreffed or none.

Reply:
I think we should cross-reference all methods. Not cross-referencing `SampleEntropy` was just forgetfulness. I noticed it while working on the `ApproxEntropy` PR and was about to submit a fix just when you commented :D